doc/ceph-volume/lvm/prepare.rst

```diff
@@ -2,25 +2,24 @@
 
 ``prepare``
 ===========
-This subcommand allows a :term:`filestore` or :term:`bluestore` setup. It is
-recommended to pre-provision a logical volume before using it with
-``ceph-volume lvm``.
+This subcommand facilitates the setup of :term:`filestore` or
+:term:`bluestore`. Before you run ``ceph-volume lvm prepare``, we recommend
+that you provision a logical volume. Then you can run ``prepare`` on that
+logical volume. 
```

Comment:
The filestore / bluestore bit is improperly capitalized, but I suggest removing as it isn’t relevant.  “of OSDs”.

Reply:
See https://github.com/ceph/ceph/pull/48665.